src/net/socket.rs

```diff
@@ -38,6 +38,24 @@ impl Socket {
         Socket::bind_with_config(addresses, Config::default())
     }
 
+    /// Bind to any local port on the system, if available
+    pub fn bind_any() -> Result<(Self, Sender<Packet>, Receiver<SocketEvent>)> {
+        let loopback = Ipv4Addr::new(127, 0, 0, 1);
```

Comment:
Now this should call `bind_any_with_config(Config::default())`

Reply:
Done